README.md

```diff
@@ -1,54 +1,104 @@
-# Kwil DB
-
-The database for Web3.
+<div align="center">
+    <h1>Kwil DB</h1>
+    <p><strong>The database for Web3.</strong></p>
+    <img src="https://github.com/kwilteam/docs/blob/main/static/img/kwil%20social-card.jpg" alt="banner" /><br/>
+    <a href="https://github.com/kwilteam/kwil-db/blob/main/LICENSE.md">
+        <img alt="GitHub License" src="https://img.shields.io/github/license/kwilteam/kwil-db">
+    </a>
+     <a href="https://github.com/kwilteam/kwil-db/releases">
+        <img src="https://img.shields.io/github/v/release/kwilteam/kwil-db" alt="Release">
+    </a>
+    <a href="https://github.com/kwilteam/kwil-db/actions">
+        <img src="https://github.com/kwilteam/kwil-db/actions/workflows/ci.yaml/badge.svg" alt="Build Status">
+    </a>
+    <a href="https://github.com/kwilteam/kwil-db/blob/main/go.mod">
+        <img src="https://img.shields.io/github/go-mod/go-version/kwilteam/kwil-db" alt="Go Version">
+    </a>
+    <a href="https://godoc.org/github.com/kwilteam/kwil-db">
+        <img src="https://godoc.org/github.com/kwilteam/kwil-db?status.svg" alt="GoDoc">
+    </a>
+    <a href="https://goreportcard.com/report/github.com/kwilteam/kwil-db">
+        <img src="https://goreportcard.com/badge/github.com/kwilteam/kwil-db" alt="Go Report Card">
+    </a>
+    <a href="https://discord.com/invite/HzRPZ59Kay">
+        <img alt="Discord" src="https://img.shields.io/discord/819855804554543114?logo=discord">
+    </a>
+</div>
+
+Kwil-db is the execution layer (node software) for Kwil Networks. Built with [PostgreSQL](https://www.postgresql.org/) and [CometBFT](https://github.com/cometbft/cometbft), Kwil-db extend the functionality of traditional relational databases to enable secure, byzantine fault tolerant, relational data-driven replicated state machines.
 
 ## Overview
 
-Learn more about Kwil at [kwil.com](https://kwil.com).
+To learn more about high-level Kwil concepts, refer to the [Kwil documentation](https://docs.kwil.com/docs/concepts).
+
+To test deploying and using a decentralized database on the Kwil testnet, refer to the [Kwil testnet tutorial](https://docs.kwil.com/docs/testnet/quickstart).
+
+For more information on kwil-db, check out the [Kwil node documentation](https://docs.kwil.com/docs/node/quickstart).
+
+## Quickstart
 
-## Build instructions
+### Build Instructions
 
-### Prerequisites
+#### Prerequisites
 
 To build Kwil, you will need to install:
 
 1. [Go](https://golang.org/doc/install)
 2. [Protocol Buffers](https://protobuf.dev/downloads/) (optional), with the `protoc` executable binary on your `PATH`.
 3. [Taskfile](https://taskfile.dev/installation)
-4. Protocol buffers go plugins and other command line tools. The `tool` task will install the required versions of the tools into your `GOPATH`, so be sure to include `GOPATH/bin` on your `PATH`.
+4. [Docker](https://docs.docker.com/get-docker/) to run a PostgreSQL database.
```

Comment:
I think maybe move docker to Local Deployment section?  We don't need docker to build kwild/kwil-cli

Reply:
I think it should probably stay here, because these are the prerequisites for quickstart (which includes postgres deployment.)